DOCUMENTATION.md

```diff
@@ -104,22 +120,174 @@ client.send(dev_id, payload, [port], [confirmation], [schedule])
 In case it's a **dictionary** with fields, please make sure the **encoder** function (Payload Formats section) of the application is set to make sense of the informations transmitted in each field.
 ![Screenshot of an encoder function in the console](./images/encoder-function.png?raw=true)
 - `port`: **int**  the port of the device to which the message will be sent. Default value to 1.
-- `confirmation`: **boolean**  this boolean indicates if you wish to receive a confirmation after sending the downlink message. Default value to False.
-- `schedule`: **string**  this string provides the type of schedule on which the message should be sent. It can take values such as `first` or `last`. Default value to `replace`.
+- `confirmation`: **boolean**  this boolean indicates if you wish to receive a confirmation after sending the downlink message. Default value to `False`.
```

Comment:
Named parameters + default values for that function + indicate it in the function signature